docs/ble.md

```diff
@@ -24,191 +37,215 @@ treat them like decimals.*
 
 Web IDL
 -------
-This IDL provides an overview of the interface; see below for documentation of
-specific API functions.
 
-```javascript
+This IDL provides an overview of the interface; see below for documentation of
+specific API functions.  We also have a short document explaining [ZJS WebIDL conventions](Notes_on_WebIDL.md).
+<details>
+<summary> Click to show/hide WebIDL</summary>
+<pre>
 // require returns a BLE object
 // var ble = require('ble');
-
-[NoInterfaceObject]
+<p><p>
```

Comment:
Here's where there are a bunch of double blanks; there are others around though.

Reply:
Geoff,
Since both you and Jimmy noticed, this, I'll go back and rip out as many as possible... :-)

Reply:
...also, I, too, thought the "<pre>" would suffice for formatting, but I just went back and checked, and it doesn't do the trick; the only thing that worked was to wrap <pre> around <details> (instead of the other way around), but then the "Details" pulldown menu is in the wrong font, which looks ugly...

I can make this switch (i.e., precede <details> with <pre>), if you guys want, but as the <p>'s are invisible to everyone except us, I'm more inclined to leave it pretty but (mildly) cumbersome. :-)

I've uploaded one file to show what moving <pre> looks like -- once Travis has its way with the docs, compare/contrast the docs for i2c.

Reply:
...also, I, too, thought the pre-construct inside of the details-construct would suffice for formatting, but I just went back and checked, and it doesn't do the trick; the only thing that worked was to wrap the pre-construct around the details-construct (instead of the other way around), but then the pulldown menu is in the wrong font, which looks ugly...

I can make this switch (i.e., put the pre-construct first), if you guys want, but as the  html-newlines are invisible to everyone except us, I'm more inclined to leave it pretty but (mildly) cumbersome. :-)

I've uploaded one file to show what swapping the two constructs looks like -- once Travis has its way with the docs, compare/contrast the docs for i2c.

Reply:
Yeah, I think the old way was better.

Reply:
Great -- I'll clean up the git record (so expect a --force'd commit :-), remove as many newlines as I can aesthetically :-), and put in a final upload.